### ISSUE TITLE
crash while booting

### ISSUE THREAD

**Issue by mango11587** [vectorwing/FarmersDelight#815]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.3

### Forge version

47.2.0

### Description

when booting game it crashes saying this "The game crashed whilst initializing game
Error: net.minecraftforge.fml.ModLoadingException: Farmer's Delight (farmersdelight) encountered an error during the common_setup event phase" tried different version and updated everything still nothing works with out the mod in 

### Steps to reproduce

boot the game

### Mod list

![image](https://github.com/vectorwing/FarmersDelight/assets/145741263/84b98b1a-d84b-4218-bf08-c399652fe7b5)
![image](https://github.com/vectorwing/FarmersDelight/assets/145741263/6443ea9d-4670-4ed7-8314-5dda144f8564)

### Logs

[logs.txt](https://github.com/vectorwing/FarmersDelight/files/12696970/logs.txt)

### Minimal instance

- [X] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by edgarogh** [comment]
Further down in the crash log, it says `Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [create.mixins.json:accessor.HumanoidArmorLayerAccessor] from phase [DEFAULT] in config [create.mixins.json] FAILED during APPLY`, which seems to emanate from Create. Try removing Create from the mod list.

The issue seems to be caused by a Create accessor mixin:
```
Caused by: org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException: No candidates were found matching renderModel(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;ILnet/minecraft/world/item/ArmorItem;Lnet/minecraft/client/model/Model;ZFFFLnet/minecraft/resources/ResourceLocation;)V in net/minecraft/client/renderer/entity/layers/HumanoidArmorLayer for create.mixins.json:accessor.HumanoidArmorLayerAccessor->@Invoker[METHOD_PROXY]::create$callRenderModel(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;ILnet/minecraft/world/item/ArmorItem;Lnet/minecraft/client/model/Model;ZFFFLnet/minecraft/resources/ResourceLocation;)V [INJECT Applicator Phase -> create.mixins.json:accessor.HumanoidArmorLayerAccessor -> Apply Accessors ->  -> Locate -> create.mixins.json:accessor.HumanoidArmorLayerAccessor->@Invoker[METHOD_PROXY]::create$callRenderModel(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;ILnet/minecraft/world/item/ArmorItem;Lnet/minecraft/client/model/Model;ZFFFLnet/minecraft/resources/ResourceLocation;)V]
```

**Comment by edgarogh** [comment]
Also duplicate of #811.

**Comment by vectorwing** [comment]
This issue seems to be fixed on Create 0.5.1f, as well as the latest release of OptiFine, HD_U_I6.

Please update Create and your performance mod, and it should be resolved. 👍
